backend/cmd/user/import.go

```diff
@@ -0,0 +1,197 @@
+package user
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/gobuffalo/pop/v6"
+	"github.com/gofrs/uuid"
+	"github.com/spf13/cobra"
+	"github.com/teamhanko/hanko/backend/config"
+	"github.com/teamhanko/hanko/backend/persistence"
+	"github.com/teamhanko/hanko/backend/persistence/models"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+func NewImportCommand() *cobra.Command {
+	var (
+		configFile string
+		inputFile  string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Import users into database from a Json file",
+		Long:  ``,
+		Run: func(cmd *cobra.Command, args []string) {
+			//Load cfg
+			cfg, err := config.Load(&configFile)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			//Load File
+			// we explicitly want user input here, hence  #nosec G304
+			jsonFile, err := os.Open(inputFile)
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer func() {
+				if err := jsonFile.Close(); err != nil {
+					log.Printf("Error closing file: %s\n", err)
+				}
+			}()
+			users, err := loadAndValidate(jsonFile)
+			if err != nil {
+				log.Fatal(err)
+			}
+			//Validate Input
+			err = validateEntries(users)
+			if err != nil {
+				log.Fatal(err)
+			}
```

Comment:
Why are the entries validated a second time? They are already validated in the `loadAndValidate` function.